provisioning/transport/mqtt/src/ProvisioningTransportHandlerMqtt.cs

```diff
@@ -180,20 +184,20 @@ private DeviceRegistrationResult ConvertToProvisioningRegistrationResult(Models.
                 try
                 {
                     if (Logging.IsEnabled) Logging.Info(this, $"Connecting to {address.ToString()}.");
-                    channel = await bootstrap.ConnectAsync(address, MqttTcpPort).ConfigureAwait(false);
+                    channel = await bootstrap.ConnectAsync(address, this.Port ?? MqttTcpPort).ConfigureAwait(false);
```

Comment:
Same as others : should always use `port`. 
Nit: remove `this.` and name the private member `_port`.